### PR TITLE
ANYRUN: Fix anyrun-get-report

### PR DESCRIPTION
**Pull request by yaakovi**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/33673

## Description
update get-report command to use request according to API docs and fallback to customer's suggestion provided [here](https://github.com/demisto/etc/issues/32803).
related PR: https://github.com/demisto/content/pull/10901

### CODE REVIEW

File: Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py [hunk 1]
```diff
@@ -779,8 +779,22 @@ def get_report(task_id):
         Response JSON from ANYRUN API call.
     """
 
-    url_suffix = 'analysis/' + task_id
-    response = http_request('GET', url_suffix=url_suffix)
+    try:
+        # according to the any-run documentation, this request should work:
+        # https://any.run/api-documentation/#api-Analysis-GetReport
+        url_suffix = 'analysis/' + task_id
```
Comment (bakatzir) [inline Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py:785]:
```suggestion
        url_suffix = f'analysis/{task_id}'
```

**Review by bakatzir — APPROVED** [review]
I see that we have no instance. please run it through the customer prior to merging.

**Comment by bakatzir** [comment]
@yaakovi Please reflect in the readme the last comment in the issue from the CS team, for future reference.

**Review by bakatzir — APPROVED** [review]
No further comments.
@amshamah419  - can you brief the README use cases and description?

**Comment by ohaim1008** [comment]
@amshamah419 @bakatzir those parts were taken from the old read me, please review the `commands` section, mostly the last line in it, it was thee only one I wrote that wasn't taken from the old readme or created automatically.

**Pull request merged**